.github/workflows/python-tests.yml

```
@@ -21,7 +21,7 @@ jobs:
           - os: ubuntu-latest
             python-version: '3.10'
             toxenv: py310-test-pytestoldest
-          - os: macos-13
+          - os: macos-14
```

maybe try latest here, too? Not exactly clear what python range is included, so just try it and revert if it doesn't work
```suggestion
          - os: macos-latest
```

---

Sure, I don't have preference. Accepted. Thanks!